### PR TITLE
CSS: Allow to pass options to style-loader and allow to remove it from loaders

### PR DESCRIPTION
**Pull request by vlad-zhukov**
~WIP because I haven't updated docs yet.~ Just wanted to get some feedback.

Changes:
- added `styleLoader` option to both `css` and `css.modules` blocks
- if it's `false` a `style-loader` won't be added
- added tests for `styleLoader` and `css.modules`

### CODE REVIEW

File: packages/assets/__tests__/css-modules.test.js [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+            localIdentName: '[name]--[local]--[hash:base64:5]',
+            modules: true
+          }
+        }
+      ]
+    }
+  ])
+})
+
+test('deprecated css(<fileType>) still works', t => {
```
Comment (andywer) [inline packages/assets/__tests__/css-modules.test.js]: I wouldn't repeat all those tests for `css.modules()`. I think it's ok to only test those backwards-compatibility stuff in one of the blocks (`css()`).

File: packages/assets/lib/css.js [hunk 2]
```diff
@@ -51,22 +57,23 @@ function cssModules (fileType, options = {}) {
...
     importLoaders: 1,
     localIdentName: String(process.env.NODE_ENV) === 'production'
-      ? '[hash:base64:10]'
-      : '[name]--[local]--[hash:base64:5]'
-  }, options)
+            ? '[hash:base64:10]'
+            : '[name]--[local]--[hash:base64:5]'
+  }, _.omit(options, ['exclude', 'include', 'styleLoader']))
+  const loaders = [{ loader: 'css-loader', options: cssLoader }]
+  if (options.styleLoader !== false) {
```
Comment (andywer) [inline packages/assets/lib/css.js]: Is this breaking change intended? Now you actually need to explicitly set `options.styleLoader` to get a `style-loader`.

I would think a vast majority of the users would like to have the style-loader by default and opt-out if necessary.

Comment (andywer) [reply packages/assets/lib/css.js]: Where do I find that in the code? Don't see it...

Comment (sapegin) [reply packages/assets/lib/css.js]: By default it’s `undefined`, which is !== `false` ;-)

Comment (andywer) [reply packages/assets/lib/css.js]: Right. It's getting late... ;P

Just wondering if we can somehow make the code a bit more speaking in this particular place.

**Comment by vlad-zhukov** [comment]
Fixed nits and updated docs.

**Comment by sapegin** [comment]
Looks like you need to update snapshots.

**Comment by vlad-zhukov** [comment]
CI is green

**Comment by andywer** [comment]
@vlad-zhukov Please don't squash the commits before pushing. We have to review everything from scratch after each push... 🙈😉

**Comment by andywer** [comment]
But thx for fixing

**Comment by andywer** [comment]
Fine for me. @sapegin Any objections?

**Comment by sapegin** [comment]
Merged :shipit:

**Comment by vlad-zhukov** [comment]
Thanks! 🎉

**Pull request merged**